packages/console/src/components/UserName/index.module.scss

```diff
@@ -4,13 +4,23 @@
   font: var(--body-medium);
   color: var(--color-text);
 
-  .userId {
+  .title {
+    display: inline-block;
+    max-width: 100%;
+    overflow: hidden;
```

Comment:
`overflow: hidden` is included in `_.text-ellipsis` mixin

Reply:
updated.